src/Jackalope/NodeType/NodeType.php

```diff
@@ -392,4 +393,28 @@ public function canRemoveProperty($propertyName, $throw = false)
 
         return true;
     }
+
+    /**
+     * @param $value
+     * @return array|mixed|string
+     */
+    private function getValueAsString($value)
+    {
+        if (is_object($value)) {
+            $val = get_class($value);
```

Comment:
can you please rewrite this to return early? this line would become `return get_class($value);`.

and we can get rid of the elseif below and just do if again.